superset/dashboards/commands/exceptions.py

```diff
@@ -51,15 +51,7 @@ def __init__(
 
 
 class DashboardCreateFailedError(CreateFailedError):
-    message = _("Dashboard could not be created.")
-
-
-class DashboardBulkDeleteFailedError(CreateFailedError):
-    message = _("Dashboards could not be deleted.")
-
-
-class DashboardBulkDeleteFailedReportsExistError(DashboardBulkDeleteFailedError):
-    message = _("There are associated alerts or reports")
+    message = _("Dashboard(s) could not be created.")
```

Comment:
To match previous exceptions:

```suggestion
    message = _("Dashboards could not be created.")
```

Reply:
@michael-s-molina see https://github.com/apache/superset/pull/24607#discussion_r1254920648. Note this will likely also help with translations.

Reply:
John, I'm not sure if I understood. My suggestion is to follow the comment you referenced. Notice that your change is in the form `Dashboard(s)` but I'm suggesting to use `Dashboards`.

Reply:
Doh.